### PR TITLE
fix: genre chips margin

### LINKED ISSUE #991: genre chips are misaligned and disappear during playback
I noticed this issue on the album page, but it might apply to other pages with the genre chips. The margin issue is probably an easy fix but I have no clue why they disappear yet.

- [x] Misalignement #995
- [ ] Disappearing

### PR DESCRIPTION
**Pull request by ThibaultNocchi**
Partially corrects #991

### CODE REVIEW

**Comment by MrTimscampi** [comment]
It looks better in the current master, imo. This PR makes the genres misaligned with everything else on the page, and create a lot of blank space.

Before
image
After
image

Before:
image
After:
image

**Comment by ThibaultNocchi** [comment]
Nice catch, I went a little bit overboard by factorizing the margin class. Will fix it tomorrow

**Comment by dkanada** [comment]
I don't know if it was mentioned, but the label should line up with the description below. It also looks like the director chips aren't aligned with the genre chips in the before image, perhaps that can be fixed as well. Maybe a component makes sense here.

**Comment by ThibaultNocchi** [comment]
@MrTimscampi @dkanada fixed, I also removed `py-sm-0` as it didn't seem to change the design. But again feel free to say otherwise haha

File: client/pages/item/_itemId/index.vue [hunk 1]
```diff
@@ -41,12 +41,9 @@
             <v-row
               v-if="item && item.GenreItems && item.GenreItems.length > 0"
               align="center"
+              class="mt-sm-3"
```
Comment (MrTimscampi) [inline client/pages/item/_itemId/index.vue:44]: We can probably remove this, as it makes the gap between the genres and the button line really large (on all pages)

**Review by MrTimscampi — APPROVED** [review]

**Review by dkanada — APPROVED** [review]

**Pull request merged**